### ISSUE TITLE
ModelAnimation added with startTime in the past isn't set to final state of animation

### ISSUE THREAD

**Issue by srcejon** [CesiumGS/cesium#10048]
If a ModelAnimation is added with a startTime in the past (i.e. startTime + duration < viewer.clock.currentTime), it appears that it is ignored, rather than having the state of the model being updated to the last frame of the animation.

Here's an example using an aircraft. When the propeller animation is started 2 seconds before the current time, the propeller isn't moved at all. If you change the startTime to the currentTime, it is.

[Sandcastle example](https://sandcastle.cesium.com/#c=jVXbjts2EP2VgR8SGdFSsr1JGl8W3e7mIUWLBnDRJwEGLY5sZinSICl7t8X+e4e62psEiV80nDlzOFf6yC0cJZ7Qwgo0nuAOnaxK9k+ti7JRXp/vjPZcarTZKIb/Mg0gdWF+M49zKLhyGAeVQ4W5l0Z/0kLm3Bt7Yd1zYU5uDt5WncZUStxqWXKPvf55vMh0potK11ywQ39L0hEbIKlc1EQcA2ov/dO4iQjgSMkcrCxlgDtKqMExl6NGNlgWA1yh3vk9QQcra3QtqDA2CkBJmHRBn2XrQ/KbN/3VX11/SUlJRHK8GLCygKi3MylgtVq16cCrV2c0UjvPdY6m6FrzpxGoLkDMIhdPF7GEn0VfWX0G4y/qeBbPcyO2n9az0gIL6rog4HNoSp2hcbJuzKoL6I5bTxLXM1ZYU97jziK6KFBdTaYzlr6/vn43+VD3/PqapW/T2fv0XX18m6YpSzPd9DzQtzXoe1efJZWQCxHVCWpe0rRko4PiGmkeg64Lat5LcSAEKEOx5l1lKiuDJ2PJmpcHhffc86SGuKRJ5lbaVtqQyHZq295AVJIKV5Wf5SOqtfyXYphMf+ls/DHY1jlXpJ9SVmlrsZUeCt7uQ7A8d6Pe5uktzx9QfOzSb+pwuQo0Cdb3bFHb8FA13l9Brj+xMYumOkkCnwo4YcM8sNDUgd8jHLjzcS2dmZx+7WnXPXgD0r92QCPCVaDwSAm3vKc9WmxIrDk4OEmlYM9poreIGqwJcNGCP7Idg9yUB976nFPS5A80lZN6V5/9yTRh/y1LWvYtKnNq6e5NCFEjihBj5UJ+5hBwcEW+3MOXynlq2gM5Sg/InaQHkLAOEYQsCoqddi6Uob62K3N/3zD8v1eKJp8GCcOErjE3WnTlZrky+QPLK0t0tV8MV9P4/J0d3KNx8z4kycubvke2GMJqs/tBVD1pDCmb/CCOYaSG1RuW7zOV5baeyk0QN5vppl+U/pr5IPamJtJ5L9W7MO4eGGre+sUo8sJTb2bUmjqJTNPoBUdT+ehyIWKY0d4FrlE8Wjr/pPCme+F+leXBEG9lVUTr75HWn5J1ybaitfMsd657npfJuetSyCNIsfrGPyHkijtHlqJS9YuQjW6WCeG/clWGC5rbv45oFX8KsP3k5o9GyRhbJnT8tqc3Rm25fcH8Pw)

This isn't a great example - as you probably wouldn't care about where a prop ends up. But if this aircraft had a landing gear animation - then it would make a very obvious difference, as the landing gear might be up instead of down.

Note that the animation is played if the animation time line is rewound. It's just not being taken in to consideration when it is initially added.

**Comment by jackbr0gm** [comment]
You are right startTime and stopTime is getting ignored. But I've tried a patchy solution like this:

[Sandcastle example](https://sandcastle.cesium.com/#c=jVZbb9s2FP4rnB9aqXEpOUnbzTcsS/KQoVsLpNiTAYOWjmyuFGmQlB2v8H/fIUVdHLdoAiTi5ZyP5/adkx3TZMdhD5rMiIQ9uQXDq5L+48+ixSDz+1slLeMS9GIwJN8WkhAuC/WHehqTggkDQ3dkQEBmuZIPMucZs0qf3G5YrvZmTKyumhNVifxG8pJZaM+P8WQhF7KopMcia7A3uNpBLYhHJqotHhKQlttDXFtEyA6d2Wpecidu0KFajpoMJNDuZtKJC5Bru0HR7pbWZ0GoUDpyghxl0gl+pkEH1xcX7dNnz59CohMRjyedLC9I1N5TnpPZbBbcIa9e9WC4NJbJDFTRpOYvlYM4EaIaWH44scX9aLCVlj0x9iyOPXuO9TJ8gmYlcygw6zkKHl1SvIfKcJ+YWWPQLdMWV0xe0UKr8g7WGsBEDurt6PKKph+ur9+PfvM5v76m6bv06kP63m/fpWlK04Wsc+7gQwza3Pk9xxCyPI+8g5KVWC2LwVYwCViP7qwxatyuhg6QkNIFa9xEptLcaVKaPLJyK+COWZZ4EZPUztxwHVZLXNK1WIUXEIpj4KryM38C8cj/QxtGl782d+zJ3T1mTOD5JXqVhhtdyS7ggQ/u5tiUevDTapZ9hfy+cb+Ogw9LjwxYC9q2eFFIeS1LvSP05EEEcrTyiXbR9fpfeAld9v6sBKYOIwEuxI+QKZk3DKOZUNlXmlVa4xtOb0hG6bDfKDr1KI5776jtS55p7Rli0F6Ay/qevaAzdAY5/hqwD9KC3jERNSGN4lAbvIg6dLo02QbySkDegYfGQOYkjWudlnCZAKZbbNOj+s8wS7aN2Gze5y6jS6HU1jec5vTYIB7r4rlM/YHbuN8kaf7WX4yuUQKoUOueVzVGkpCHguyhLobOQGw1xG6AbJmxQ7/qXRn52rr4EasIt68Nwb7AhIOwQBobyH4DGmoQrbaG7LkQZMOwja0AJNHKiedB+J6uKclUuWVBpw+J7a6DqQyXa7+3e9XVsEFUofYB7k45EyVA7mysDHRF+BZ1mSX/VsYiU7+iIrcEmOE49VDWAJCcFwXajo3WhcE/28b2jfvby2Pbibpe9BkVbnw9Ld1yubxctn2jtXfcLdur2sJxu/LZrfP0Jumy23IXieNKvwWiOTv8XZUr9OSid2pqfn0q7thh0odQ2xahfvAZQDg80z/pB0p+4fjBONzvkGgfubE4YHXLKRJ5uXYg/YwDBRfIHE8DRl1I/TT8QVRjT5nVbN4yJkkEFqZrUsG3s6514uT57Zm3oR+cSZJ5L/RuBJ9LTLvQxg3Ut/7cj1aB3r+gj6N41ZJ91JEd5xH8WCntKfU6ROgPdfH4z7GeHkiOx2dUZwUGnFyR4PlCIrWdyaqy0emMGZIrHGbxZDAcTI09CJg3D/7Oy61C1EqLCCeqBZyoSF2TrCqcZJZmxjRNa5r0Vac53xGez77zzyUGlBmDN0Ul/JBdDObTBOXPVIViOXaFTzvQgh2c2GY0/1gfUkqnCW6/r2mVEiumnyH/Dw)
